spec/providers/https.spec.ts

```diff
@@ -164,6 +164,7 @@ async function runTest(test: CallTest): Promise<any> {
 }
 
 // MockRequest mocks an https.Request.
+// tslint:disable-next-line
```

Comment:
what's the reason for this?

Reply:
Rule that allows only one class per file - I didn't want to change this here yet, but perhaps I should leave the `tslint:disable` off so that we can fix that in the future. We are not currently running linter on tests so it won't fail.

Reply:
Actually I just moved this out into its own file in `fixtures` and now we don't need that disable line.